src/core/server/http/https_redirect_server.test.ts

```diff
@@ -91,8 +91,7 @@ test('throws if [redirectHttpFromPort] is in use', async () => {
     } as HttpConfig)
   ).rejects.toMatchSnapshot();
 
-  // Workaround for https://github.com/DefinitelyTyped/DefinitelyTyped/issues/17605.
-  (mockListen as any).mockRestore();
+  mockListen.mockRestore();
```

Comment:
seems this is a common way in our codebase to restore mocks, but imho it's incorrect. this code is unreachable if assertion fails. am I right? should I create another issue to fix all such cases? @joshdover

Reply:
Yeah that's a good catch. Could push the mock into an array that gets cleaned up in an `afterEach` block. Up to you whether or not you want to fix this now or separately.